### DISCUSSION TITLE
Updating image to latest causes container to go unhealthy

### DISCUSSION THREAD

**Discussion by kosymodo** [qdm12/gluetun#2257] [Q&A]
It's been a few weeks since I last updated my gluetun image.  Attempted to update it today, and it caused the container to go unhealthy.  Error message 'no iptables supported found'.  Reverting image to v3.37.0 resolves the issue.  Any ideas why v3.38.0 would cause this problem?  It's not exactly a new release.  Any help would be very gratefully received!

**Comment by Shadex9** [comment]
I am having the same issue therefor I made a [report.](https://github.com/qdm12/gluetun/issues/2260)
BTW for me a rollback to version V3.38 worked

**Reply by kosymodo** [reply]
@Shadex9 what I don’t understand from your reply/report is that v3.38.0 is the latest, so how are you rolling back to this version? Unless you mean that :latest doesn’t work, but manually specifying v3.38.0 does?

**Reply by Shadex9** [reply]
@kosymodo Indeed the latest tag does not work, V3.38 tag does work. They are not the same V3.38 is the stable release

**Comment by kosymodo** [comment]
All now sorted with latest code - reverting to latest version tag works
